src/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -374,6 +389,19 @@ public boolean match(final Territory t) {
     };
   }
 
+  public static Match<Territory> territoryHasNonMobileInfraFactoryAndIsNotConqueredOwnedLand(final PlayerID player,
```

Comment:
Nope. Updated and simplified `territoryHasNonMobileInfraFactoryAndIsLand `to `territoryHasNonMobileInfraFactory ` since its only used in 1 place that already checks isLand.